scss/prototype/_display.scss

```diff
@@ -0,0 +1,62 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display classes
+/// @type Boolean
+$prototype-display-breakpoints: $global-prototype-breakpoints !default;
+
+/// Map containing all the `display` classes
+/// @type Map
+$prototype-display-classes: (
+  none, inline, inline-block, block, table, table-cell
+) !default;
+
+/// Map containing all the `display-flex` classes
+/// @type Map
+$prototype-display-flex-classes: (
+  flex, inline-flex
+) !default;
+
+@mixin foundation-prototype-display {
+  @each $display in $prototype-display-classes {
+    .display-#{$display} {
+      display: $display !important;
```

Comment:
Ohk @IllusionPerdu ..... I get your idea..... I will look into it .... thanks !

Also can you help me in above comment :arrow_double_up:  ...... https://github.com/zurb/foundation-sites/pull/9738#issuecomment-277518057